Source/WebCore/html/HTMLElement.h

```diff
@@ -148,7 +148,8 @@ class HTMLElement : public StyledElement {
     void addHTMLPixelsToStyle(MutableStyleProperties&, CSSPropertyID, StringView value);
     void addHTMLNumberToStyle(MutableStyleProperties&, CSSPropertyID, StringView value);
 
-    void addHTMLColorToStyle(MutableStyleProperties&, CSSPropertyID, const String& color);
+    std::optional<SRGBA<uint8_t>> parseLegacyColorValue(StringView);
```

Comment:
I suggest we make this a static member function.

Reply:
That's a good point. Will do.